packages/grid/data-grid/src/DataGrid.test.tsx

```diff
@@ -606,7 +606,7 @@ describe('<DataGrid />', () => {
       }
     });
 
-    it('should increase grid density by 50% when selecting compact density', () => {
+    it('should increase grid density when selecting compact density', () => {
```

Comment:
Well now you need to move those test in another file, maybe toolbar test or density?

Reply:
Damien is making a reference to #722 that was merged on HEAD. We need to rebase.

Reply:
Saw that, I've resolved the conflicts ;)